test/common/http/async_client_impl_test.cc

```diff
@@ -273,8 +274,8 @@ TEST_F(AsyncClientImplTracingTest, BasicNamedChildSpan) {
   AsyncClient::RequestOptions options = AsyncClient::RequestOptions()
                                             .setParentSpan(parent_span_)
                                             .setChildSpanName(child_span_name_)
-                                            .setSampled(false);
-  EXPECT_CALL(*child_span, setSampled(false));
+                                            .setSampled(absl::nullopt);
+  EXPECT_CALL(*child_span, setSampled(_)).Times(0);
```

Comment:
Can we reserve this test case where set sampling status to `false` and just add a new case where set sampling status to `absl::nullopt`?

Reply:
Of course, good point! Done.